frontend/vue/components/Syllabus/SyllabusForm.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <section class="syllabus-form">
+    <bx-tabs :value="defaultActiveTab">
+      <bx-tab id="tab-write" class="syllabus-form__tab" target="panel-write" value="write">
+        {{ $translate('Write') }}
+      </bx-tab>
+    </bx-tabs>
+    <div id="panel-write" class="syllabus-form__tab-panel" role="tabpanel" aria-labelledby="tab-write">
+      <SyllabusFormCourseInfo />
+      <SyllabusFormModule
+        v-for="module in modules"
+        :key="module.id"
+        :module-id="module.id"
+        :module-count="modules.length"
+        @removeModuleAction="removeContentBlock(module.id)"
+      />
+      <div class="syllabus-form__add-content">
+        <BasicLink
+          class="syllabus-form__add-content__link"
+          v-bind="addContentLink"
+          @click="addContentBlock"
+        >
+          {{ addContentLink.label }}
+        </BasicLink>
+      </div>
+      <SyllabusInlineNotification
+        v-if="showInlineNotification"
+        @on-close-notification="closeNotification"
+        @undo-deletion="undoDeleteAction(lastDeletedModuleId)"
+      />
```

Comment:
I think so, yes... but this can also come in a later PR.

Maybe @gracelindsell @pandasa123 might have input as well:

What should the user see if they delete multiple module/content blocks?

- 1 single inline notification telling the user that they deleted a Syllabus section/module
- multiple, stacked inline notifications telling the user that they deleted multiple Syllabus sections/modules
- something else